src/components/ConfirmItem/components/ConfirmItemList.js

```diff
@@ -0,0 +1,130 @@
+  [CONFIRM_ITEM_RECORD_FIELD_NAME.MEDIATED_REQUEST_STATUS]: { max: 180 },
+  [CONFIRM_ITEM_RECORD_FIELD_NAME.REQUESTER]: { max: 120 },
+  [CONFIRM_ITEM_RECORD_FIELD_NAME.REQUESTER_BARCODE]: { max: 120 },
+};
+export const getConfirmItemColumnMapping = (confirmItemType) => {
+  const COLUMN_MAPPING = CONFIRM_ITEM_RECORD_TRANSLATIONS;
+
+  COLUMN_MAPPING[CONFIRM_ITEM_RECORD_FIELD_NAME.DATE] = confirmItemType === CONFIRM_ITEM_TYPES.CONFIRM_ITEM_ARRIVAL
+    ? CONFIRM_ITEM_RECORD_TRANSLATIONS[CONFIRM_ITEM_RECORD_FIELD_NAME.ARRIVAL_DATE]
+    : CONFIRM_ITEM_RECORD_TRANSLATIONS[CONFIRM_ITEM_RECORD_FIELD_NAME.IN_TRANSIT_DATE];
+
+  return omit(COLUMN_MAPPING, [CONFIRM_ITEM_RECORD_FIELD_NAME.ARRIVAL_DATE, CONFIRM_ITEM_RECORD_FIELD_NAME.IN_TRANSIT_DATE]);
+};
+export const getConfirmItemListFormatter = (confirmItemType) => ({
+  [CONFIRM_ITEM_RECORD_FIELD_NAME.DATE]: (confirmItem) => {
+    const datePath = confirmItemType === CONFIRM_ITEM_TYPES.CONFIRM_ITEM_ARRIVAL
+      ? CONFIRM_ITEM_RECORD_FIELD_PATH[CONFIRM_ITEM_RECORD_FIELD_NAME.ARRIVAL_DATE]
+      : CONFIRM_ITEM_RECORD_FIELD_PATH[CONFIRM_ITEM_RECORD_FIELD_NAME.IN_TRANSIT_DATE];
+
+    return (
+      <AppIcon
+        size="small"
+        app={APP_ICON_NAME}
+      >
+        <FormattedTime
+          value={get(confirmItem, datePath, DEFAULT_VIEW_VALUE)}
+          day="numeric"
+          month="numeric"
+          year="numeric"
+        />
+      </AppIcon>
+    );
+  },
+  [CONFIRM_ITEM_RECORD_FIELD_NAME.TITLE]: (confirmItem) => (
+    get(confirmItem, CONFIRM_ITEM_RECORD_FIELD_PATH[CONFIRM_ITEM_RECORD_FIELD_NAME.TITLE], DEFAULT_VIEW_VALUE)
+  ),
+  [CONFIRM_ITEM_RECORD_FIELD_NAME.ITEM_BARCODE]: (confirmItem) => (
+    get(confirmItem, CONFIRM_ITEM_RECORD_FIELD_PATH[CONFIRM_ITEM_RECORD_FIELD_NAME.ITEM_BARCODE], DEFAULT_VIEW_VALUE)
+  ),
+
+  [CONFIRM_ITEM_RECORD_FIELD_NAME.EFFECTIVE_CALL_NUMBER]: (confirmItem) => {
+    const item = get(confirmItem, CONFIRM_ITEM_RECORD_FIELD_PATH[CONFIRM_ITEM_RECORD_FIELD_NAME.ITEM], {});
+
+    return effectiveCallNumber(item);
+  },
+  [CONFIRM_ITEM_RECORD_FIELD_NAME.MEDIATED_REQUEST_STATUS]: (confirmItem) => (
+    get(confirmItem, CONFIRM_ITEM_RECORD_FIELD_PATH[CONFIRM_ITEM_RECORD_FIELD_NAME.MEDIATED_REQUEST_STATUS], DEFAULT_VIEW_VALUE)
+  ),
+  [CONFIRM_ITEM_RECORD_FIELD_NAME.REQUESTER]: (confirmItem) => (getRequesterName(confirmItem)),
+  [CONFIRM_ITEM_RECORD_FIELD_NAME.REQUESTER_BARCODE]: (confirmItem) => (
+    get(confirmItem, CONFIRM_ITEM_RECORD_FIELD_PATH[CONFIRM_ITEM_RECORD_FIELD_NAME.REQUESTER_BARCODE], DEFAULT_VIEW_VALUE)
+  ),
+});
+export const getEmptyMessage = (contentData) => (
+  contentData.length
+    ? null
+    : <FormattedMessage id="ui-requests-mediated.confirmItem.mainSection.listItems.isEmptyMessage" />
+);
+
+const ConfirmItem = ({
+  confirmItemType,
+  contentData,
+}) => {
+  return (
+    <MultiColumnList
+      id="confirmItemList"
+      data-testid="confirmItemList"
```

Comment:
seems like you are not using this test id. Would you like to clear it?

Reply:
Fixed (removed  data-testid="confirmItemList")